ssg/build_stig.py

```diff
@@ -7,14 +7,7 @@
 from .constants import XCCDF11_NS, stig_ns, stig_refs
 
 
-def add_references(input_file_name, reference_file_name):
-    """
-    For a given input XCCDF file and DISA STIG reference XCCDF file, process
-    all STIG references in the rules from input XCCDF file and correctly link
-    them to the corresponding reference rule.
-
-    Returns the updated ElementTree containing updated reference elements.
-    """
+def get_versions(reference_file_name):
```

Comment:
The name is misleading, the function returns a mapping of version strings (versions of what?) to rule IDs.